src/swap/faast.js

```diff
@@ -316,19 +317,24 @@ export function makeFaastPlugin(opts: EdgeCorePluginOptions): EdgeSwapPlugin {
         toCurrencyCode
       )
 
-      const spendTarget: EdgeSpendTarget = {
+      const spendTarget = {
         nativeAmount: quoteFor === 'to' ? fromNativeAmount : nativeAmount,
-        publicAddress: quoteData.deposit_address
+        publicAddress: quoteData.deposit_address,
+        otherParams: {
+          uniqueIdentifier: quoteData.deposit_address_extra_id
+        }
       }
 
-      const spendInfo: EdgeSpendInfo = {
```

Comment:
Why remove the types?

Reply:
@dylanseago any resolution on the types? I'm ready to test and merge this

Reply:
Sorry, this is a non-issue. I forgot to resolve